### PR TITLE
add yaqd-system-monitor

### PR DESCRIPTION
**Pull request by untzag**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by untzag** [comment]
@ksunden could you comment confirming they are willing to be a maintainer

**Comment by untzag** [comment]
OK, I'm now realizing that anaconda cloud doesn't contain [uptime](https://pypi.org/project/uptime/), marking this as WIP now and will come back to this package :smile:

**Comment by ksunden** [comment]
I am willing to maintain

File: recipes/yaqd-system-monitor/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  number: 0
+  noarch: python
+  entry_points:
+    - yaqd-system-monitor = yaqd_system_monitor._system_monitor:SystemMonitor.main
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
```
Comment (ksunden) [inline recipes/yaqd-system-monitor/meta.yaml:23]:
```suggestion
    - python >=3.7
```

File: recipes/yaqd-system-monitor/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  run:
+    - psutil
+    - python >=3.8
```
Comment (ksunden) [inline recipes/yaqd-system-monitor/meta.yaml:26]:
```suggestion
    - python >=3.7
```

**Comment by untzag** [comment]
@conda-forge/help-python this is ready for review---thanks :smile:

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**